.github/workflows/release.yml

```diff
@@ -1,42 +1,67 @@
-name: publish_release
+name: release
 
 on:
   push:
     tags:
       - 'v*'
 
 jobs:
-  goreleaser:
-    if: ${{ github.ref == 'refs/heads/master' }}
-    runs-on: ubuntu-latest
+  build-mac:
+    runs-on: macos-latest
     steps:
       -
         name: Code checkout
         uses: actions/checkout@v2
       -
-        name: Run goreleaser
-        uses: ./.github/actions/releaser
+        name: Set up Go
+        uses: actions/setup-go@v2
         with:
-          args: '--rm-dist'
-        env:
-          GORELEASER_GITHUB_TOKEN: ${{ secrets.GORELEASER_GITHUB_TOKEN }}
-          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
-      -
-        name: Upload to Bintray
-        run: |
-          scripts/publish-deb-to-bintray.sh
-          scripts/publish-rpm-to-bintray.sh
-        shell: bash
+          go-version: 1.16
+      -
+        name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v2
```

Comment:
seems like this is all using the correct go v of 1.16 _but_ I thought we were supposed to use goreleaser-xcgo to allow cgo compilation for some mac-related issues (dns related I think)? Did you confirm that we can now use goreleaser directly and everything works? (@suz-stripe has more context)

Reply:
The `xcgo` setup was needed predominantly because we were compiling a mac-native binary on Linux, which required some different tooling. If we run the mac-native compilation on mac-* platforms, it should be fine moving forward!